datarobot_batch_scoring/main.py

```diff
@@ -252,12 +254,18 @@ def main(argv=sys.argv[1:]):
     skip_row_id = parsed_args['skip_row_id']
     output_delimiter = parsed_args.get('output_delimiter')
 
+    if pid and not lid:
```

Comment:
Maybe we keep the current format with 3 positional args and we make a special string for PID,  for example:
```batch_scoring --host=http://localhost ... SSE  586d5807c80891461e6423c7 path/to/dataset.csv``` Where `"SSE"` is a special string that that indicates a stand-alone prediction 

That would look like ```batch_scoring --host=<host> --user=<user> {<project_id>| "SSE" } <model_id>  <dataset_filepath> --datarobot_key=<datarobot_key> {--password=<pwd> | --api_token=<api_token>}```